### PR TITLE
[KO] Translation of deepdream.ipynb

### PR DESCRIPTION
**Pull request by jaketae**
Supersedes #104

### CODE REVIEW

**Comment by jaketae** [comment]
@rickiepark, thank you so much for the detailed feedback. I have applied your comments and made the necessary edits as per your suggestions. I know I complicated things by messing up Git and reopening a new issue, but I hope this is okay... Please let me know if you find any issues!

File: site/ko/tutorials/generative/deepdream.ipynb [hunk 1]
```diff
@@ -0,0 +1,726 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "sb7u31B4NbGt"
+      },
+      "source": [
+        "## 손실 계산하기\n",
+        "\n",
+        "손실은 선택한 층들의 활성화값의 총 합으로 계산됩니다. 층의 크기와 상관 없이 모든 활성화값이 동일하게 고려될 수 있도록 각 층의 손실을 정규화합니다. 일반적으로, 손실은 경상하강법으로 최소화하고자 하는 수치입니다. 하지만 딥드림에서는 예외적으로 이 손실을 경사상승법(gradient ascent)을 통해 최대화할 것입니다."
```
Comment (rickiepark) [inline site/ko/tutorials/generative/deepdream.ipynb:282]: '경상상승법' --> '경사상승법'

File: site/ko/tutorials/generative/deepdream.ipynb [hunk 2]
```diff
@@ -0,0 +1,726 @@
...
+        "\n",
+        "지금 생성된 이미지도 상당히 인상적이지만, 위의 시도는 몇 가지 문제점들을 안고 있습니다: \n",
+        "\n",
+        "  1. 생성된 이미지에 노이즈(noise)가 많이 끼어있습니다 (이 문제는 [`tf.image.total_variation loss`](https://www.tensorflow.org/api_docs/python/tf/image/total_variation)로 해결할 수 있습니다).\n",
+        "  1. 생성된 이미지의 해상도가 낮습니다.\n",
+        "  1. 패턴들이 모두 균일한 입도(granularity)로 나타나고 있습니다.\n",
+        "  \n",
+        "이 문제들을 해결할 수 있는 한 가지 방법은 경상상승법의 스케일(scale)를 달리하여 여러 차례 적용하는 것입니다. 이는 작은 스케일에서 생성된 패턴들이 큰 스케일에서 생성된 패턴들에 녹아들어 더 많은 디테일을 형성할 수 있도록 해줍니다.\n",
+        "\n",
+        "이 작업을 실행하기 위해서는 이전에 구현한 경상상승법을 사용한 후, 이미지의 크기(이를 옥타브라고 부릅니다)를 키우고 여러 옥타브에 대해 이 과정을 반복합니다.\n"
```
Comment (rickiepark) [inline site/ko/tutorials/generative/deepdream.ipynb:464]: Two '경상상승법' --> '경사상승법'

**Comment by jaketae** [comment]
@rickiepark Fixed the typo! I'll try to be more meticulous with my translations next time. Thank you so much again!

**Comment by rickiepark** [comment]
Great!
Thanks for you efforts.
LGTM 👍

**Comment by lamberta** [comment]
Fixed merge conflict and ran nbfmt, so should be good now

**Comment by jaketae** [comment]
Thank you so much to both!

**Pull request merged**